src/analysis/inner_product_space/dual.lean

```diff
@@ -67,22 +67,13 @@ show ∥(to_dual_map 𝕜 E).to_continuous_linear_map∥ = 1,
 
 variable (𝕜)
 include 𝕜
+
 lemma ext_inner_left {x y : E} (h : ∀ v, ⟪v, x⟫ = ⟪v, y⟫) : x = y :=
-begin
-  apply (to_dual_map 𝕜 E).map_eq_iff.mp,
-  ext v,
-  rw [to_dual_map_apply, to_dual_map_apply, ←inner_conj_sym],
-  nth_rewrite_rhs 0 [←inner_conj_sym],
-  exact congr_arg conj (h v)
-end
+by rw [←sub_eq_zero, ←inner_self_eq_zero, inner_sub_right, sub_eq_zero, h (x - y)]
 
 lemma ext_inner_right {x y : E} (h : ∀ v, ⟪x, v⟫ = ⟪y, v⟫) : x = y :=
```

Comment:
I guess the golfed way to do this using `to_dual` is
```lean
(to_dual_map 𝕜 E).injective $ continuous_linear_map.coe_fn_injective $ funext h
```
But your observation is, why bring `to_dual` into it at all?  Fair enough.

In that case, could you please move the lemmas to `inner_product_space/basic`?